### PR TITLE
[swift5] fix demo app

### PR DESCRIPTION
**Pull request by michaelknoch**
**Type of change:** fix

## Motivation (current vs expected behavior)
fixes the demoapp

## Please check if the PR fulfills these requirements
- [x] Self-review: I am confident this is the simplest and clearest way to achieve the expected behaviour
- [ ] There are no dependencies on other PRs or I have linked dependencies through Zenhub
- [ ] The commit messages are clean and understandable
- [ ] Tests for the changes have been added (for bug fixes / features)

### CODE REVIEW

File: samples/getting-started/cmake_caches.cmake [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Copied from NDK's android.toolchain.cmake:
```
Comment (michaelknoch) [inline samples/getting-started/cmake_caches.cmake]: i think this file could exist in `swift-android-toolchain` @ephemer

File: samples/getting-started/go.sh [hunk 2]
```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```
Comment (michaelknoch) [inline samples/getting-started/go.sh]: we could also move a more abstract version of this build script to the toolchain or at least uikit

File: samples/getting-started/cmake_caches.cmake (hunk 1 above)
Comment (ephemer) [reply samples/getting-started/cmake_caches.cmake]: Yeah it probably should

File: samples/getting-started/go.sh (hunk 2 above)
Comment (ephemer) [reply samples/getting-started/go.sh]: yeah I don't see any reason not to put this into the toolchain too!

File: samples/getting-started/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.15.0)
```
Comment (ephemer) [inline samples/getting-started/CMakeLists.txt]: This should be pretty much the only thing that needs to change here :)

**Pull request merged**